simple_history/models.py

```diff
@@ -173,7 +173,8 @@ def copy_fields(self, model):
                 if getattr(old_field, 'db_column', None):
                     field_arguments['db_column'] = old_field.db_column
                 field = FieldType(
-                    old_field.rel.to,
+                    # required for Django <= 1.8                     # required for Django >= 2.0
+                    old_field.rel.to if hasattr(old_field, 'rel') else old_field.remote_field.model,
```

Comment:
Having this inline makes for a dense line IMHO.

Reply:
Suggest, for readability

```
if hasattr(old_field, 'rel'):
    target = old_field.rel.to  # Django <= 1.8 
else:
    target = old_field.remote_field.model # Django >= 2.0
field = FieldType(target,
    ...

```

Reply:
This could even be determined just once and saved (in this module before this class, or in a shared compat module).

Reply:
Also I think the test should be for the new attribute name; both exists in django 1.11 but the old-style one causes a warning.